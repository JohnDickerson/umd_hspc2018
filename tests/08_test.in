13
1
3
2
4
6
8
20
30
40
50
60
100
200
