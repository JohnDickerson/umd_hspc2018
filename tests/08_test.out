0
0
229
7728
2069
4990
5205
4701
8473
825
5498
446
7751